handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -331,6 +341,14 @@ public static boolean supportsKeyManagerFactory() {
         return SUPPORTS_KEYMANAGER_FACTORY;
     }
 
+    /**
+     * Returns {@code true} if <a href="https://wiki.openssl.org/index.php/Hostname_validation">Hostname Validation</a>
+     * is supported when using OpenSSL.
+     */
+    public static boolean supportsHostnameVerification() {
```

Comment:
Just a nit but openssl calls it "Hostname Validation" while we call "Hostname Verification". Maybe make it consistent ?